esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -763,18 +763,23 @@ static void filterXmlBySchema(IPTree* in, IXmlType* type, const char* tag, Strin
     }
 }
 
-static void filterXmlBySchema(StringBuffer& in, StringBuffer& schema, StringBuffer& out)
+static void filterXmlBySchema(StringBuffer& in, StringBuffer& schema, const char* qName, StringBuffer& out)
 {
     Owned<IXmlSchema> sp = createXmlSchema(schema);
     Owned<IPTree> tree = createPTreeFromXMLString(in);
 
     //VStringBuffer name("tns:%s", tree->queryName());
-    const char* name = tree->queryName();
+    const char* name = qName;
```

Comment:
You could have just passed in "name" and skipped this assignment.  Even name = tree->queryName() below is ok because you are only changing what name points to.